ddtrace/_logger.py

```diff
@@ -0,0 +1,66 @@
+import logging
+from logging.handlers import RotatingFileHandler
+import os
+
+from ddtrace.internal.utils.formats import asbool
+
+
+DEFAULT_FILE_SIZE_BYTES = 15 << 20  # 15 MB
+
+
+def configure_ddtrace_file_logger():
+    # type: () -> None
+    """Configures ddtrace log levels and file paths.
+
+    Customization is possible with the environment variables:
+        ``DD_TRACE_DEBUG``, ``DD_TRACE_LOG_FILE_LEVEL``, and ``DD_TRACE_LOG_FILE``
+
+    By default, when none of the settings have been changed, ddtrace loggers
+        inherit from the root logger in the logging module and no logs are written to a file.
+
+    When DD_TRACE_DEBUG has been enabled:
+        - Propagate logs up so logs show up in the application logs if a file path wasn't provided
+        - Logs are routed to a file when DD_TRACE_LOG_FILE is specified, using the log level in DD_TRACE_LOG_FILE_LEVEL.
+        - Child loggers inherit from the parent ddtrace logger
+
+    Note(s):
+        1) The ddtrace-run logs under commands/ddtrace_run do not follow DD_TRACE_LOG_FILE if DD_TRACE_DEBUG is enabled.
+            This is because ddtrace-run calls ``logging.basicConfig()`` when DD_TRACE_DEBUG is enabled, so
+            this configuration is not applied.
+        2) Python 2: If the application is using DD_TRACE_DEBUG=true, logging will need to be configured,
+            ie: ``logging.basicConfig()``.
+
+    """
+    debug_enabled = asbool(os.environ.get("DD_TRACE_DEBUG", "false"))
+
+    log_file_level = os.environ.get("DD_TRACE_LOG_FILE_LEVEL", "DEBUG")
+    try:
+        file_log_level_value = getattr(logging, log_file_level.upper())
+    except AttributeError:
+        raise ValueError(
+            "Unknown log level. Logging level must be CRITICAL/ERROR/WARNING/INFO/DEBUG. Detected logging level was ",
```

Comment:
```suggestion
            "Unknown log level. Logging level must be CRITICAL/ERROR/WARNING/INFO/DEBUG.",
```

We probably don't need this trailing part, it should be clear from the way `ValueError` is formatted (is it...?)